Sources/DNA/Font/Font.swift

```diff
@@ -34,66 +37,78 @@ public extension UIFont {
         registerCustomFonts()
 
         let font = UIFont(name: FontType.light.rawValue, size: 28.0)!
-        return font.scaledFont(forTextStyle: .title2)
+        return font.scaledFont(forTextStyle: .title1)
```

Comment:
What's the reason for changing this .title1 to .title2?

Reply:
https://developer.apple.com/ios/human-interface-guidelines/visual-design/typography/